fermipy/defaults.py

```diff
@@ -101,7 +101,6 @@ def make_default_dict(d):
     'resample': (True, '', bool),
     'srcmap': (None, '', str),
     'bexpmap': (None, '', str),
-    'wmap': (None, 'Likelihood weights map.', str),
```

Comment:
git should automatically handle merging changes in the dev and master branches.  

What seems to have happened in this case is that you made your changes against older versions of `gtanalysis.py` and `defaults.py` and then committed those back into the lightcurve-dev branch.  I don't see how this could have happened unless you were editing the file somewhere else and then subsequently copied it back into the git repository.  In general before making commits I would always run `git diff` to double-check that the changes you're committing are what you expect.  Maybe your text editor was cacheing an older version of the file somehow?  Are you using emacs?